direct/io-kafka/src/main/java/cz/o2/proxima/direct/kafka/KafkaAccessor.java

```diff
@@ -117,6 +130,7 @@ public KafkaAccessor(EntityDescriptor entity, URI uri, Map<String, Object> cfg)
     this.cfg = cfg;
     this.topic = Utils.topic(uri);
     configure(cfg);
+    this.adminClient = createAdmin();
```

Comment:
Ehm, I might have taken you on wrong path. I investigated and there is currently no lifecycle method for disposing this, so please return this back to the version where the client is created for each `isAcceptable` call (when it is state commit log). Thanks.